### DISCUSSION TITLE
Why the multiple class names be merged into a single one

### DISCUSSION THREAD

**Discussion by alixiashu** [microsoft/fluentui#28818] [General] [answered]
# Behavior
- Use the `mergeStyleSets` to generate 2 class names
- Use these 2 class names in a single component, like the button in my example
- The results would be
    - The styles defined in both classes take effect
    - But when I check the element class names in the "Elements" tab of dev tool, the only latter class name I write is reserved.

# Repro
I wrote a [sandbox](https://codepen.io/alixiashu/pen/rNQEMwg) for this feature.
There are 2 class names defined, which are a & b, both take effect. 
![image](https://github.com/microsoft/fluentui/assets/28770220/ef8afcb1-36ec-431a-ba76-0b443f43b593)

But in the "Elements" tab, only "b" is left in the class name list.
![image](https://github.com/microsoft/fluentui/assets/28770220/e68eb45d-3e11-4c8c-b326-fcb8a305c0ab)

**Comment by ValentinaKozlova** [comment, marked as answer]
It looks like you might want to use here `mergeClasses`. While `mergeStyleSets` focuses on combining style objects, `mergeClasses` is used to combine CSS class names generated by styling hooks.
